src/index.ts

```diff
@@ -1590,7 +1589,29 @@ export class BigQuery extends common.Service {
       // The Job is important for the `queryAsStream_` method, so a new query
       // isn't created each time results are polled for.
       options = extend({job}, options);
-      job!.getQueryResults(options, callback as QueryRowsCallback);
+
+      // table#getRows uses listTableData endpoint, which is a faster method
+      // to read rows of the results. However, it won't work for model queries,
+      // so use the original job#getQueryResults for model queries.
+      if (
+        typeof query === 'string' &&
+        (query.includes('CREATE MODEL') || query.includes('REPLACE MODEL'))
```

Comment:
Might want to use a regex here since these could be lowercase. Is there nothing available to us in the response body that would indicate we created a model?

Reply:
Thanks for the catch! Totally meant to do that!